### ISSUE TITLE
Nullness checker:  constructor that does not  initialize all fields

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#74]
```
The Nullness Checker does not issue an error if not all fields have been
initialized by the time the constructor exits.

A test case for this is inner class RawAfterConstructor in file
checker-framework/checkers/tests/nullness/RawTypes.java .

```

Original issue reported on code.google.com by `michael.ernst@gmail.com` on 28 Jul 2010 at 4:21
